content/tutorial/tutorial.md

```diff
@@ -56,9 +55,9 @@ The tutorial is divided into several sections:
 
 हा प्रारंभ करण्याचा वेगवान मार्ग आहे!
 
-प्रथम हे नवीन टॅबमध्ये उघडा  **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** नवीन टॅबमध्ये रिक्त टिक-टॅक-टू गेम बोर्ड आणि प्रतिक्रिया कोड प्रदर्शित करेल. या ट्यूटोरियल मध्ये आपण रिऍक्ट कोड संपादित करू.
+प्रथम हे नवीन टॅबमध्ये उघडा  **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** नवीन टॅबमध्ये रिक्त टिक-टॅक-टू गेम बोर्ड आणि React कोड प्रदर्शित करेल. या ट्यूटोरियल मध्ये आपण React कोड संपादित करू.
```

Comment:
टिक-टॅक-टू =>  टिक-टैक्-टो

Please change this to wherever use

Reply:
Done!